src/engine.rs

```diff
@@ -149,6 +149,9 @@ pub struct Reedline {
     // Manage optional kitty protocol
     kitty_protocol: KittyProtocolGuard,
 
+    // Echo typed input
```

Comment:
As this an absolutely obscure usecase I think expanding this to a small doccomment with pointers what you are trying to achieve and the relation to the `termios` flag makes sense for future generations.